### PR TITLE
Feature/create event from map screen

### PR DESCRIPTION
**Pull request by pauromeu**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/cook4me/ui/map/MapScreen.kt [hunk 1]
```diff
@@ -88,6 +90,7 @@ fun GoogleMapView(
             .fillMaxWidth()
             .padding(vertical = 16.dp, horizontal = 16.dp)
     ) {
+        CreateNewItemButton(itemType = "Event", onClick = onCreateNewEventClick)
```
Comment (dayan9265) [inline app/src/main/java/ch/epfl/sdp/cook4me/ui/map/MapScreen.kt:93]: I think the "Event" should also be in the string resource

**Review by dayan9265 — APPROVED** [review]
Except the small detail of some strings that should be in the resource (you can directly merge after this small change), the code looks nice and it is a good idea to have to have created a generic add button 👍

File: app/src/main/java/ch/epfl/sdp/cook4me/ui/map/MapScreen.kt (hunk 1 above)
Comment (pauromeu) [reply app/src/main/java/ch/epfl/sdp/cook4me/ui/map/MapScreen.kt]: Yes, it should. Changed!

File: app/src/main/java/ch/epfl/sdp/cook4me/ui/common/button/CreateNewItemButton.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                imageVector = Icons.Default.Add,
+                contentDescription = "Create a new $itemType",
+                modifier = Modifier.size(24.dp)
+            )
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(text = "Create a new $itemType")
```
[the remark this answers is not in the document]
Comment (pauromeu) [reply app/src/main/java/ch/epfl/sdp/cook4me/ui/common/button/CreateNewItemButton.kt]: Already changed. Thanks!

**Pull request merged**